partiql-logical/src/lib.rs

```diff
@@ -1,13 +1,24 @@
 use partiql_value::{BindingsName, Value};
+use petgraph::prelude::StableGraph;
+use petgraph::Directed;
 use std::collections::HashMap;
 
-// TODO we should replace this enum with some identifier that can be looked up in a symtab/funcregistry?
 #[derive(Debug)]
+pub struct LogicalPlan(pub StableGraph<BindingsExpr, (), Directed>);
```

Comment:
Not to worry while we iterate, but we'll want to remove petgraph from the exposed logical plan soon and replace something like a simple list of nodes and list of edges.